newIDE/app/src/MainFrame/ElectronMainMenu.js

```diff
@@ -95,7 +95,10 @@ class ElectronMainMenu extends React.Component<Props, {||}> {
     );
     ipcRenderer.on(
       ('main-menu-create': MainMenuEvent),
-      event => this._editor && this._editor.openCreateDialog()
+      event =>
+        this._editor &&
+        this._editor.askToCloseProject() &&
```

Comment:
I think this might be a good thing to do but we should do another PR for this :)